src/Moryx.Products.Model/Moryx.Products.Model.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Database model for the ProductManagement module</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>MORYX;IIoT;IoT;DataModel;Model;Database;Npgsql;PostgreSQL;Postgres</PackageTags>
```

Comment:
I would remove "DataModel;Model;Database;Npgsql;PostgreSQL;Postgres" and add instead the same tags like for the ProductManagement